geo/src/main/java/com/github/davidmoten/geo/LatLong.java

```diff
@@ -61,4 +61,21 @@ public String toString() {
         return builder.toString();
     }
 
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        final LatLong latLong = (LatLong) o;
+        return Double.compare(lat, latLong.lat) == 0 && Double.compare(lon, latLong.lon) == 0;
```

Comment:
Why not normal equality checks?

Reply:
Don't worry about it, probably a good idea in the face of NaNs etc.

Reply:
The answer was that I didn't have strong opinions here. So I let IntelliJ autogenerate it on the hopes that it would generate idiomatic code. We saw with `hashcode()` that it has some misses. 😓 
I think actual comparison of the doubles would work even if one is `Float.NaN`; `Float.NaN == Float.NaN` resolves to `true`. But `Double#compare(double, double)` seems to have some edge cases to compare, for example, -0 and 0.